crypto/tss/eddsa/frost/signer/round_1.go

```diff
@@ -245,20 +253,27 @@ func (p *round1) Finalize(logger log.Logger) (types.Handler, error) {
 		return nil, ErrTrivialSignature
 	}
 	p.c, err = SHAPoints(p.pubKey, R, p.message)
+
 	if err != nil {
 		return nil, err
 	}
 	p.r = R
-
 	// Compute own zi = di+ ei*li + c bi xi
 	selfNode := p.nodes[p.peerManager.SelfID()]
+	if err != nil {
```

Comment:
The previous line doesn't generate an error ?

Reply:
done!